filebeat/module/mongodb/log/test/mongodb-debian-3.2.11.log-expected.json

```diff
@@ -6,11 +6,11 @@
         "event.module": "mongodb",
         "fileset.name": "log",
         "input.type": "log",
+        "log.level": "I",
```

Comment:
That does not really follow our field convention but don't have a better suggestion at the moment.

Reply:
Well there's no normalization on this yet. The examples in ECS right now are WARN, ERR and INFO... I think this should be the raw value.

Reply:
And these are equivalent to log levels, btw. [doc](https://docs.mongodb.com/manual/reference/log-messages/#severity-levels) expected values are F for fatal, E for error, W for warning, I for informational and D for debug.